docs/sources/enterprise/license-restrictions.md

```diff
@@ -35,6 +35,12 @@ Sometimes it is useful to sign in to an account from multiple locations simultan
 
 Given a limit of three sessions, the longest inactive session is signed out of when a fourth person signs in to the same account.
 
+### Usage billing
+
+You can turn on usage billing in Enterprise licensing in order to have an unlimited number of active users. When usage billing is enabled, Grafana does not enforce active user limits. Instead, we charge for active users above the limit, according to your customer contract. 
```

Comment:
I'm not sure the latest "You can" change is semantically correct, as that is not an action taken by the customer. Wdyt? 👇

```suggestion
You can request usage billing in Enterprise licensing in order to have an unlimited number of active users. When usage billing is enabled, Grafana does not enforce active user limits. Instead, we charge for active users above the limit, according to your customer contract. 
```

Reply:
Yeah good point, this though sounds a bit odd
> You can request usage billing in order to have an unlimited number of active users

What about
>You can request usage billing to be turned on in order to have an unlimited number of active users

and add a link to wherever they can request it.

Reply:
cc @oddlittlebird what do you think about the suggestion?

Reply:
Works for me. I just try to replace the "it is possible" construction wherever I see it.

Reply:
Another suggestion:
"You can request Grafana Labs to enable billing to allow an unlimited number of active users ..."

Reply:
There have been multiple discussions around the wording "turn on" vs "enable". I honestly lost at one point what was the consensus, but I know that in the UI we shod the flag as "On" or "Off".  So perhaps mentioning "to be turned on" would give better context to customers about what we are talking about.

Reply:
@achatterjee-grafana I've incorporated your feedback, hope this looks good now. If there are no objections I'd like to get this merged today.

Thanks everyone for contributions! 💯